src/vswhere.lib/Formatter.cpp

```diff
@@ -495,6 +495,7 @@ HRESULT Formatter::GetInstanceId(_In_ ISetupInstance* pInstance, _Out_ VARIANT*
     return GetStringProperty(bind(&ISetupInstance::GetInstanceId, pInstance, _1), pvtInstanceId);
 }
 
+#pragma warning(suppress: 6101) // pvtInstallDate is not set if value is empty but we still return success
```

Comment:
Nit: and variant should be empty. That's what the defualt ctor for `variant_t` [does](https://docs.microsoft.com/en-us/cpp/cpp/variant-t-variant-t?view=msvc-160#:~:text=_variant_t(%20)%20Constructs%20an%20empty%20_variant_t%20object%2C%20VT_EMPTY.).

Only a nit because the comment isn't wrong, but tells an incomplete story for future maintainers. The result is an empty variant that just means "I didn't find it, but it's no big deal in most cases" because this information wasn't always available, for example. Callers can still decide to err if it's `VT_EMPTY` on return, though.